### PR TITLE
add README.md for Snips.ai addon

### PR DESCRIPTION
**Pull request by Mofeywalker**
Add README.md for Snips.ai addon. References #738

### CODE REVIEW

File: snips/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+Now it's time to start Snips for the first time. You can configure the microphone and sound card using the Add-on interface. Now start the add-on.
+
+## Configuration
+
+Add-on configuration:
+
+```json
+{
+  "mqtt_bridge": {
```
Comment (frenck) [inline snips/README.md]: This is not available (according to the schema: <https://github.com/home-assistant/hassio-addons/blob/master/snips/config.json#L23>)

File: snips/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# Hass.io Core Add-on: Snips.AI
+
+Support for Snips.ai voice assistant.
+
+![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield]
```
Comment (frenck) [inline snips/README.md]: Add all architectures to the list, use coloring (red/green) to indicate if it is supported or not.

File: snips/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+# Hass.io Core Add-on: Snips.AI
+
+Support for Snips.ai voice assistant.
+
+![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield] 
+## About
+
+Snips.ai is an AI-powered voice assistant that runs on the Raspberry Pi 3 and x86 platforms.
+In contrast to Google Assistant or Amazon Alexa it runs on-device and is Private by Design.
```
Comment (frenck) [inline snips/README.md]:
```suggestion
In contrast to Google Assistant or Amazon Alexa, it runs on-device and is private by design.
```

File: snips/README.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  "custom_tts": {
+      "active": false,
+      "platform": "amazon_polly"
+  }
+}
+```
+
+### Option: `assistant`
+
+The name of your custom assistant in ```/share```. If no assistant is found then a default assistant will be used.
```
Comment (frenck) [inline snips/README.md]:
```suggestion
The name of your custom assistant in `/share`. If no assistant is found, then a default assistant will be used.
```

File: snips/README.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+### Option: `language`
+
+This is used to select the default custom assistant. Currently ```en```, ```de``` and ```fr``` are supported.
```
Comment (frenck) [inline snips/README.md]:
```suggestion
This is used to select the default custom assistant. Currently, `en`, `de` and `fr` are supported.
```

File: snips/README.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+which custom tts. 
+
+#### Option `custom_tts.active`
+
+Indicates whether a custom tts is used or not.
+
+#### Option: `custom_tts.platform`
+
+Specifies which TTS platform to use.
+
```
Comment (frenck) [inline snips/README.md]: Only a single empty line needed.

Comment (Mofeywalker) [reply snips/README.md]: fixed in 7bbc54012f9507e05558cdfa552baae923f901dc

File: snips/README.md (hunk 1 above)
Comment (Mofeywalker) [reply snips/README.md]: fixed in 5099828f70aa224d356a6b29edc6de7eed016fc6

File: snips/README.md (hunk 2 above)
Comment (Mofeywalker) [reply snips/README.md]: fixed in f9843167e364d49bcc634300ddd3aaa65bef48e0

**Pull request merged**